### PR TITLE
Fix memory leaks in tests from the presto-tests module

### PR DESCRIPTION
**Pull request by arhimondr**
Close query runners after test classes are done

### CODE REVIEW

**Comment by haozhun** [comment]
@electrum needs to review this.

**Comment by haozhun** [comment]
In addition, this didn't seem to solve the Travis failure. Maybe something is trying to use as much as memory as there is available.

**Comment by arhimondr** [comment]
Yeah, i noticed that. I'm going to figure out what test causes that.

**Comment by arhimondr** [comment]
I just fixed the memory leaks, and it seems it shouldn't fail anymore. But let's just wait for the build to finish.

**Comment by findepi** [comment]
@arhimondr please see https://github.com/prestodb/presto/issues/10812#issuecomment-405248405 and subsequent comment for info about memory leak. Your PR doesn't seem to fix this particular leak. Was there more?

File: presto-tests/src/test/java/com/facebook/presto/execution/TestBeginQuery.java [hunk 1]
```diff
@@ -80,12 +81,19 @@ public void setUp()
         getQueryRunner().createCatalog("tpch", "tpch", ImmutableMap.of());
     }
 
-    @BeforeMethod
-    public void beforeMethod()
+    @AfterMethod
```
Comment (findepi) [inline presto-tests/src/test/java/com/facebook/presto/execution/TestBeginQuery.java]: `alwaysRun=true` (highly recommended, even if not strictly necessary in this particular case)

**Comment by arhimondr** [comment]
I'm not 100% sure, but i think it was coming out of the `TestMemoryAwareExecution`

**Comment by arhimondr** [comment]
merged

**Pull request closed without merging**